css/main.css

```diff
@@ -195,6 +196,22 @@ td, th {
   text-align: center;
 }
 
+input[type=submit]
+{
+    border: 2px;
+    border-radius:4px;
+    width:250px;
+    padding: 16px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 20px;
+    margin: 4px 2px;
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
```

Comment:
Actually I changed the color of the button on hover later removed it . For that only I have used transition-duration